CMakeLists.txt

```diff
@@ -477,9 +477,29 @@ IF(NOT ASPECT_ADDITIONAL_CXX_FLAGS STREQUAL "")
   MESSAGE(STATUS "  DEAL_II_CXX_FLAGS_RELEASE: ${DEAL_II_CXX_FLAGS_RELEASE}")
 ENDIF()
 
+# Check if a fastscape library is available, if so link it with aspect.
+# If the library is not found, then remove fastscape .cc and .h from list
+# so aspect can still compile.
+# TODO: likely need to switch ENV and normal positions so it checks ENV second.
```

Comment:
I would make the files always compile but fail at runtime if it can not be used. You would introduce a cmake define ASPECT_WITH_FASTSCAPE.